website/contributing/release-troubleshooting.md

```diff
@@ -3,6 +3,53 @@ id: release-troubleshooting
+   1. open the **Road to 0.X.0** discussion (e.g: [Road to 0.71.0](https://github.com/reactwg/react-native-releases/discussions/41)) or the **Should we release 0.X.Y** discussion (e.g.: [Should we release 0.70.7](https://github.com/reactwg/react-native-releases/discussions/48))
+   2. edit the main message adding the link to your commit under the **Local commits to backport to main** section
+
+Once the issue in the `-stable` branch is handled, make sure to backport the fixes to main. When the PR backporting the fix is open against `main`, make sure to update the relevant release discussion by adding the link to the PR right below the local commit in the **Local commits to backport to main** section. This makes it easier to keep track that the local commit has been taken care of.
+
+Typically, the author of the original change should backport it. In that way, the commit's attribution is preserved. If the author is not available (for example, they are in PTO or in a different timezone and the backporting is urgent to unblock other work), anyone can take care of it.
+
+### The Git Tag Has Been Pushed to Remote but CI Failed to Release to NPM
+
+The release process is composed of various steps, of which two are key for the rollout:
+
+1. Create a version tag on the GitHub repository
+2. Publish all the artifacts on Maven and NPM
+
+Step 2 takes most of the CI time because it has to build all the artifacts and prefabs needed. It might happen that CI will fail during this part, not reaching the publish to NPM step.
+
+In case Step 1 completes successfully but Step 2 fails, the code in the stable branch is set to the new version, but it has failed to rollout. This means that we need to do some extra steps to be able to retry to publish this version.
+If you end up in this situation, follow these steps:
+
+1. Remove the tag that has been pushed (update the `X`, `Y` and `k` values appropriately):
+
+```sh
+git tag -d v0.X.Y[-RC.k]
+git push origin :v0.X.Y[-RC.k]
+```
+
+1. Revert the commit pushed by Step 1. The script we use for the release updates the version numbers all across the codebase in many files ([here's an example](https://github.com/facebook/react-native/commit/38465f2d184a2558681a6b6b45163694c885bd39)). The commit usually has this title `[0.X.Y-rc.k] Bump version numbers` for RCs or `[0.X.Y] Bump version numbers` for stable releases.
+
+```sh
+git revert <sha of the commit>
+git push origin 0.X-stable
+```
+
+1. Apply all the necessary fixes in the `-stable` branch of the codebase. The remedies for this could be of two types:
+   - you forgot to cherry-pick some commits. In this case, you follow the standard cherry-pick procedure for RCs and patches.
+   - you need to fix some bugs in the code. See the ["Add a local commit"](#adding-a-local-commit-in-a--stable-branch) section below.
+1. Once you have pushed all the necessary commits in the branch, re-trigger the release process **with the same tag** via the usual script.
```

Comment:
in the preview deployment it looks like markdown is not able to keep the numeration: https://deploy-preview-3433--react-native.netlify.app/contributing/release-troubleshooting#the-git-tag-has-been-pushed-to-remote-but-ci-failed-to-release-to-npm

can you hardcode the steps numbers?